plutus-metatheory/src/Utils.lagda.md

```diff
@@ -177,6 +177,7 @@ postulate ByteString : Set
 
 postulate
   eqByteString : ByteString → ByteString → Bool
+  mkByteString : String → ByteString
```

Comment:
do you need a `COMPILE` pragma?

Reply:
I don't think so. `mkByteString` is only used for generating the certificates which are only meant to be type-checked, and not compiled down to Haskell.